### PR TITLE
Partial fix for 1830

### LINKED ISSUE #1830: UI auth tweaks for authentication workflow
A few changes to make for auth launch with annotations that Colin and I discussed to free up space in main window and keep all info and account management in the left upper corner:

- [x] Remove door icon from log in button
- [x] Move log in and info buttons from the top bar to in line with the cellxgene icon and dataset name
- [x] Hover over on login button should say "Log in to cellxgene"
- [x] Once a user logs in, "log in" button should be removed. Instead the drop down from the info button should say "Logged in as {email_address}" and show a "log out" option.

Ping @colinmegill for any design questions.

### PR DESCRIPTION
**Pull request by colinmegill**
* Remove door icon from log in button
* Move log in and info buttons from the top bar to in line with the cellxgene icon and dataset name
* Hover over on login button should say "Log in to cellxgene"
* Show email

closes #1830

### CODE REVIEW

File: client/src/components/menubar/infoMenu.js [hunk 1]
```diff
@@ -58,9 +64,16 @@ const InformationMenu = React.memo((props) => {
                 text="Privacy Policy"
               />
             ) : null}
+            {userinfo.is_authenticated ? (
```
Comment (bmccandless) [inline client/src/components/menubar/infoMenu.js:67]: "Log Out"  should be shown if the authentication type "requires_client_login" and the user "is_authenticated".

The way this is written the "Log Out" menu item would be shown when the server is configured for session based authentication.  In session based (which is the default), the user is automatically logged in, and there is no way to logout, so the Log Out button should not be shown.

**Review by bmccandless — COMMENTED** [review]
Looks good.  I found one issue when the session based authentication is used.

File: client/src/components/menubar/infoMenu.js (hunk 1 above)
Comment (colinmegill) [reply client/src/components/menubar/infoMenu.js:67]: Nice catch! Thanks, will change

**Review by bmccandless — APPROVED** [review]
LGTM

File: client/src/components/menubar/infoMenu.js [hunk 2]
```diff
@@ -1,31 +1,37 @@
...
             <MenuItem
               onClick={() => handleClick(dispatch)}
-              icon={IconNames.BOOK}
+              icon="info-sign"
               text="Dataset Overview"
             />
 
             <MenuItem
               href="https://chanzuckerberg.github.io/cellxgene/"
               target="_blank"
```
Comment (lesliecodes) [inline client/src/components/menubar/infoMenu.js:32]: sorry, this is existing code, but we should be adding rel="noopener" to any new tab links for security.

https://web.dev/external-anchors-use-rel-noopener/

**Review by bmccandless — APPROVED** [review]
LGTM.  I tested with authentication types = oauth, session, and none.  All work as expected.

**Comment by lesliecodes** [comment]
Fixed the merge conflict and added rel="noopener" to the tab links.
Looks ok still: 👀 
image

File: client/src/components/menubar/infoMenu.js [hunk 3]
```diff
@@ -3,48 +3,58 @@ import React from "react";
...
+    dispatch,
+  } = props;
   return (
     <ButtonGroup className={`${styles.menubarButton}`}>
       <Popover
         content={
           <Menu>
             <MenuItem
               onClick={() => handleClick(dispatch)}
-              icon={IconNames.INFO_SIGN}
```
Comment (seve) [inline client/src/components/menubar/infoMenu.js:27]: @colinmegill Why drop the use of bp's enum?

**Pull request merged**